sample/src/main/res/layout/main_act.xml

```diff
@@ -14,20 +14,28 @@
   ~  limitations under the License.
   -->
 
-<androidx.coordinatorlayout.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<RelativeLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Why replacing the CoordinatorLayout with a RelativeLayout?

Reply:
Relative Layout in my experience loads better when you have a number of components stacked into the layout so used it to add the toolbar at top.